packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -264,6 +290,14 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
             });
         }
 
+        if (initializeEntryPoint) {
+            this.entryPoint = new FluidObjectHandle<FluidObject>(
+                new LazyPromise(async () => initializeEntryPoint(this)),
+                "",
+                this.objectsRoutingContext,
+                );
```

Comment:
nit: spacing.

Reply:
Rewrote these lines to make them easier to parse.